backend/src/models/expense.ts

```diff
@@ -0,0 +1,10 @@
+export type Expense = {
+  expense_id: number;
+  category_id: number;
+  amount: number;
+  Expense_date: Date;
+  summary: string | null;
+};
+
+export type CreateExpensePayload = Omit<Expense, 'expense_id' | 'category_id'>;
```

Comment:
Used for when sending a post request to create a expense, the payload has some attributes of category not all. Omit just takes the type and removes the fields that you specify